presto-docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -64,6 +87,14 @@ Constructors
 
     Returns a geometry type polygon object from WKT representation.
 
+.. function:: to_spherical_geography(Geometry) -> SphericalGeography
+
+    Converts a geometry value defined in (longitude, latitude) space to a geometry on the sphere of the Earth's radius.
+
+.. function:: from_spherical_geography(SphericalGeography) -> Geometry
```

Comment:
I just realized that, perhaps, a better name would be `to_geometry(SphericalGeography)`. This name describes the function completely: take a spherical geometry and turn it into a geometry. `from_spherical_geography` name only describes the input, but doesn't describe the output. 

In the future, one might want to add a Geography type and a conversion function from SphericalGeography to Geography. They could do this by adding `to_geography(SphericalGeography)` function, but they wouldn't be able to overload `from_spherical_geography(SphericalGeography)` function.